src/main/java/com/CezaryZal/api/day/DayController.java

```diff
@@ -1,67 +1,81 @@
+import com.CezaryZal.api.day.entity.day.DayBasic;
+import com.CezaryZal.api.day.manager.DayApiService;
+import com.CezaryZal.api.day.manager.DayService;
 import io.swagger.annotations.Api;
 import io.swagger.annotations.ApiOperation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
 import javax.security.auth.login.AccountNotFoundException;
-import java.util.List;
 
 @CrossOrigin(origins = "*", maxAge = 3600)
 @Api(tags = "Day")
 @RestController
 @RequestMapping("/api/day")
 public class DayController {
 
-    private DayService dayS;
+    private final DayService dayService;
+    private final DayApiService dayApiService;
 
-    public DayController(DayService DService) {
-        this.dayS = DService;
-    }
-
-    @ApiOperation(value = "This will get a `Day` by id")
-    @GetMapping("/{id}")
-    public Day getDayById(@PathVariable Long id){
-        return dayS.getDayById(id);
-    }
-
-    @ApiOperation(value = "This will get a `DayDTO` by date and user id")
-    @GetMapping("/dto/{date}/{userId}")
-    public DayDTO getDayDTOByDateAndUserId(@PathVariable String date, @PathVariable Long userId) throws AccountNotFoundException {
-        return dayS.getDayDTOByDateAndUserId(date, userId);
+    @Autowired
+    public DayController(DayService dayService, DayApiService dayApiService) {
+        this.dayService = dayService;
+        this.dayApiService = dayApiService;
     }
 
     @ApiOperation(value = "This will get a day id by date and user id")
     @GetMapping("/day-id/{date}/{userId}")
-    public Long getDayIdByDateAndUserId(@PathVariable String date, @PathVariable Long userId){
-        return dayS.getDayIdByDateAndUserId(date, userId);
+    public ResponseEntity<Long> getDayIdByDateAndUserId(
+            @PathVariable String date,
+            @PathVariable Long userId) {
+        return new ResponseEntity<>(dayService.getDayIdByDateAndUserId(date, userId), HttpStatus.OK);
     }
 
-    @ApiOperation(value = "This will get a `Day` by date and user id")
-    @GetMapping("/{date}/{userId}")
-    public Day getDayByDateAndUserId(@PathVariable String date, @PathVariable Long userId){
-        return dayS.getDayByDateAndUserId(date, userId);
+    @ApiOperation(value = "This will get a basic information of `DayApi` by date and user id")
+    @GetMapping("/api/basic/{date}/{userId}")
+    public ResponseEntity<DayApi> getDayApiByDateAndUserId(
+            @PathVariable String date,
+            @PathVariable Long userId){
+        long currentTime = System.currentTimeMillis();
+        ResponseEntity<DayApi> dayApiResponseEntity = new ResponseEntity<>(dayApiService.getDayApiByDateAndUserId(date, userId), HttpStatus.OK);
+        Long processTime = System.currentTimeMillis() - currentTime;
+        System.out.println(processTime);
```

Comment:
Ten print ma zostać w kodzie? Jeżeli tak to od tego jest logger.
Możesz skorzystać z lombokowego, jeżeli już z niego korzystasz.

Reply:
Zapomniałem usunąć.
Sprawdzałem na szybko wydajnościowe kwestie